.github/workflows/mirror-calico.yml

```diff
@@ -0,0 +1,57 @@
+name: Sync GHCR Calico images with Docker Hub
+on:
+  schedule:
+    # run every 12h
+    - cron:  '0 */12 * * *'
+  workflow_dispatch:
+
+jobs:
+  get-docker-release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Login to GitHub Container Registry (ghcr)
+        run: echo ${{ secrets.GHCR_PASSWORD }} | docker login ghcr.io -u ${{ secrets.GHCR_USERNAME }} --password-stdin
```

Comment:
Where do these secrets come from? The other way to do this would be to link the packages to this repository and then specify permissions on the default github token for the workflow.

Reply:
This secret needs to be manually added from the GH Admin settings page.

> specify permissions on the default github token for the workflow.

IMHO, I would prefer to split privileges in case we have a token leak.

Reply:
ok